### PR TITLE
Editor secuirty page mvp

### PR DESCRIPTION
**Pull request by Gabrn**
Initial Security page in Editor.
screen shot 2018-12-22 at 18 58 28

### CODE REVIEW

**Comment by Yshayy** [comment]
Looks good :)
Can you had a screenshot to the PR?

**Comment by Yshayy** [comment]
https://g.codefresh.io/public/accounts/soluto/builds/5c30ef1be9274f30a66a334a

**Comment by Yshayy** [comment]
also react-editable-json-tree

**Comment by Gabrn** [comment]
docker build should pass now..

File: services/editor/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "jwt-decode": "2.2.0",
     "lunr": "^2.1.6",
     "moment": "^2.22.0",
-    "oidc-client": "1.5.4",
+    "oidc-client": "1.4.1",
```
Comment (michaelkruglos) [inline services/editor/package.json]: Downgrade?

File: services/editor/src/pages/security/components/SecurityPage/PoliciesEditor.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <EditableTable
+      data={unsavedPolicies}
+      onEditTable={onEdit}
+      columns={{
+        user: 'string',
+        group: 'string',
+        contexts: 'json',
+        object: 'string',
+        action: 'string',
+        effect: 'yes-or-no',
```
Comment (michaelkruglos) [inline services/editor/src/pages/security/components/SecurityPage/PoliciesEditor.js]: Does it produce allow/deny strings?

File: services/editor/src/pages/security/components/SecurityPage/index.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  withProps,
+  withHandlers,
+  branch,
+} from 'recompose';
+import Rx from 'rxjs';
+import { getPolicies, putPolicies } from '../../../../store/ducks/policies';
+import './index.css';
+import PoliciesEditor from './PoliciesEditor';
+
+const objects = [
```
Comment (michaelkruglos) [inline services/editor/src/pages/security/components/SecurityPage/index.js]: Is it used?

File: services/editor/vapid/keys.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"publicKey":"BHuIV94-d6F1C45UllomBuDHg8f3ZnCFdUcnZgqjsES-R76WN8_GJGavb1ViluGyacGCiQvaFQXNtfGDICBnuXk","privateKey":"FLYJXfxvNLhRHHjdP-vOhj_5BLGAnsJKpqrKPDPxSi4"}
```
Comment (michaelkruglos) [inline services/editor/vapid/keys.json]: I don't think you're supposed to commit this file

File: services/editor/src/pages/security/components/SecurityPage/PoliciesEditor.js (hunk 2 above)
Comment (Gabrn) [reply services/editor/src/pages/security/components/SecurityPage/PoliciesEditor.js]: It is for allow/deny. But in the security page it appears as an on/off switch

**Pull request closed without merging**